source/npm/qsharp/src/compiler/compiler.ts

```diff
@@ -204,7 +204,7 @@ export class Compiler implements ICompiler {
 
   async getQir(program: ProgramConfig): Promise<string> {
     return callAndTransformExceptions(async () =>
-      this.wasm.get_qir(toWasmProgramConfig(program, "base")),
+      this.wasm.get_qir(toWasmProgramConfig(program, "adaptive_rif")),
```

Comment:
I was about to suggest that this default match the behavior of VS Code and Jupyter where "unrestricted" is the default, but then I realized that might have the side effect of breaking QCOM since this is down in the npm layer. That said, even this change will make QCOM start producing adaptive code rather than base, right?

Reply:
The VS Code default for the "generate QIR" command is adaptive_rif, so this does match that.

Reply:
The idea is that, because this is qir generation, the assumed behavior should be the minimal required, which here is adaptive_rif. This matches VS Code behavior, when the user tries to generate qir locally.

Reply:
Changing this back to what it was before to not risk breaking changes to dependents. The default for the playground will be set higher up the stack.